.github/workflows/CI.yml

```diff
@@ -64,7 +64,6 @@ jobs:
       run: cargo hack check --each-feature --no-dev-deps --all
 
   test-versions:
```

Comment:
I think you can remove `stable` from this job's `matrix` because the `test-examples` job will run all of the regular tests as well.

Reply:
Moved things around a bit in 2decbe357b41f